src/NuGet.Core/NuGet.Packaging/PackageCreation/Resources/NuGetResources.resx

```diff
@@ -212,4 +212,8 @@
     <value>The element 'icon' cannot be empty.</value>
     <comment>Please, don't localize icon</comment>
   </data>
+  <data name="InvalidPlatformVersion" xml:space="preserve">
+    <value>Target Framework '{0}' has a platform component, but the platform could not be inferred. This is likely a problem with NuGet itself.</value>
```

Comment:
```suggestion
    <value>Target Framework '{0}' has a platform component, but the platform could not be inferred.</value>
```

If a customer does `<None Include="whatever.txt" PackagePath="lib/net5.0-windows/" />`, then this isn't an internal issue at all.